tinywallet/tinywallet/screens.py

```diff
@@ -1103,6 +1103,16 @@ def __init__(self):
         self.layout.addWidget(wgt)
         grid.setColumnStretch(0, 1)
         row = 0
+        grid.addWidget(Q.makeLabel("Relay Fee", 14, Q.ALIGN_LEFT), row, 0)
+        row += 1
+        self.relayFeeField = QtWidgets.QLineEdit(str(app.settings[DB.relayFee]))
+        lbl = Q.makeLabel(" atoms / kb ", 14)
```

Comment:
Field is pretty wide for the few digits needed. Fiddle with the width of the field, `self.relayFeeField.setFixedWidth`, and the grid layout. You may have to use the optional `columnSpan` argument of `QGridLayout.addWidget` to get things to look right.

Reply:
I actually think just three choices would be better for a user. With the few wallet apps I have used it is so. You can choose from the normal fee or more. Is having a lower fee even a good idea I wonder? Anyway, I will get some data from the blockchain and share before continuing.